__tests__/app.test.js

```diff
@@ -97,13 +93,14 @@ describe('GET /api/articles/:article_id/comments', () => {
         .get("/api/articles/3/comments")
         .expect(200)
         .then(({body})=> {
-            expect(Array.isArray(body)).toBe(true);
-            expect(body).toHaveLength(2);
-            expect(body).toBeSortedBy("created_at", {
+            const comments = body.comments;
+            expect(Array.isArray(comments)).toBe(true);
+            expect(comments).toHaveLength(2);
+            expect(comments).toBeSortedBy("created_at", {
                 descending: true,
             });
-            if(body.length > 0){
-                body.forEach((comment) => {
+            if(comments.length > 0){
```

Comment:
There's no need for the if statement here, as it will already fail at the expect statement on line 98 if it's en empty array.